src/plugins/embeddable/public/lib/state_transfer/types.ts

```diff
@@ -17,33 +17,49 @@
  * under the License.
  */
 
+import { EmbeddableInput } from '..';
+
 /**
  * Represents a state package that contains the last active app id.
  * @public
  */
-export interface EmbeddableOriginatingAppState {
+export interface EmbeddableEditorState {
   originatingApp: string;
+  byValueMode?: boolean;
```

Comment:
as discussed, I'd rather if we didn't have an explicit "byValue" parameter, but totally fine with leaving it for now.